assets/js/source.js

```diff
@@ -110,19 +110,20 @@ export function scrollBottom() {
 }
 
 async function logTemplate(e) {
-  const {
-    via_rule,
-    origin_source_id,
-    body
-  } = e
-  const metadata = JSON.stringify(body, null, 2)
-  const formatter = await userSelectedFormatter()
-  const formattedDatetime = formatter(body.timestamp)
-  const randomId = Math.random() * 10e16
-  const metadataId = `metadata-${body.timestamp}-${randomId}`
-
-  const metadataElement = !_.isEmpty(body.metadata) ?
-    `
+  const { via_rule, origin_source_id, body } = e;
+  const metadata = JSON.stringify(body, null, 2);
+  const formatter = await userSelectedFormatter();
+  const formattedDatetime = formatter(body.timestamp);
+  const randomId = Math.random() * 10e16;
+  const metadataId = `metadata-${body.timestamp}-${randomId}`;
+  const log_level = _.get(body, ["metadata", "level"]);
```

Comment:
small note: can refactor in future to import the specific module instead to reduce bundle size

Reply:
Sure - though I'm hoping we can just replace the bulk of this with LiveView.